services/api-server/src/simcore_service_api_server/api/dependencies/models_schemas_function_filters.py

```diff
@@ -0,0 +1,33 @@
+from typing import Annotated, Any
+
+from fastapi import Query
+from models_library.functions import FunctionJobCollectionsListFilters
+from pydantic.fields import FieldInfo
+
+
+def _get_query_params(field: FieldInfo) -> dict[str, Any]:
```

Comment:
I think this was also somewhere,. right?
perhap s we can move it to some `/dependencies/_utils.py`?

Reply:
Yeah, i got it from your solver file. I'll move it to _utils